tests/testthat/helper-tests.R

```diff
@@ -2,7 +2,40 @@ test_path <- function(context, path) {
   file.path("examples", context, path)
 }
 
+cache_con <<- NULL
```

Comment:
I don't believe you want the double arrow here, just within the functions.  What I usually do though is:

```
cache <- new.env(parent = emptyenv())
```

Then use this as

```
if (is.null(cache$con)) {
  cache$con <- new_connection()
}
cache$con
```

etc

Reply:
Thanks - have gone for the better way!